src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/gc/shared/GCCause.java

```diff
@@ -60,11 +62,11 @@ public enum GCCause {
   _z_allocation_rate ("Allocation Rate"),
   _z_allocation_stall ("Allocation Stall"),
   _z_proactive ("Proactive"),
+  _z_high_usage ("High Usage"),
```

Comment:
gcCause.hpp has the Shenandoah and ZGC blocks swapped. Could you fix it in this change as well?

Reply:
OK, I have swapped the Shenandoah and ZGC blocks here.